cmd/printer/interactive_unix.go

```diff
@@ -0,0 +1,7 @@
+//go:build !windows
```

Comment:
is this required? Given we're using the `_unix` and `_windows` suffixes, AFAIK, the compiler will pick the required files

Reply:
Apparently, from what I read, the selection based on filename works as follows: `filename_GOOS_GOARCH.go` so for the `_windows.go` we can skip it, indeed. However `_unix.go` doesn't seem to be a valid suffix, as per `go tool dist list`. So I shall remove the build tag for windows, but the one you pointed as to be kept, AFAIU.